tox.ini

```diff
@@ -43,10 +43,13 @@ commands =
 
 [testenv:render-machine]
 description = Render the machine charm
-allowlist_externals = cp
+allowlist_externals =
+    cp
+    chmod
 commands =
-    cp src/machine_charm.py src/charm.py
-    cp machine_metadata.yaml metadata.yaml
+    cp {toxinidir}/src/machine_charm.py {toxinidir}/src/charm.py
+    chmod +x {toxinidir}/src/charm.py
+    cp {toxinidir}/machine_metadata.yaml {toxinidir}/metadata.yaml
```

Comment:
like charm.py is gitignored, should we also gitignore metadata.yaml?

Reply:
(and have a separate k8s-metadata.yaml?)

Reply:
That is exactly what we do. Unfortunately, part of the CI (I don't remember which part) requires a stub metadata.yaml file to exist. That stub being checked in "overrides" gitignore :(